auth/commondetails.go

```diff
@@ -233,11 +247,11 @@ func SshTokenRefreshPreRequestInterceptor(fields *CommonConfigFields, httpClient
 	return nil
 }
 
```

Comment:
In the *SshTokenRefreshPreRequestInterceptor* function in this class, the following comment should be probably changed:
// Obtain a new token and return true (false for error). --> // Obtain a new token.
(because a boolean is not returned there).